src/DevBetterWeb.Core/Interfaces/ApplicationUser.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace DevBetterWeb.Core.Interfaces
+{
+  internal class ApplicationUser
```

Comment:
What is this doing in the interfaces namespace?

Reply:
no idea. Probably accidentally added it when using user manager<applicationUser> somewhere